cidc_api/services/ingestion.py

```diff
@@ -84,7 +83,14 @@ def extract_schema_and_xlsx() -> Tuple[str, str, BinaryIO]:
 
 
 @ingestion_api.route("/validate", methods=["POST"])
-@resource("ingestion/validate")
+@requires_auth(
+    "ingestion/validate",
```

Comment:
if it's actually just for manifest validation, should it disallow `CIMAC_BIOFX_USER` then?